source/grammar/index.rst

```diff
@@ -3,6 +3,19 @@ OpenQasm 3.0 Grammar
 
 OpenQasm 3.0 Grammar specification based in ANTLR_ parser generator.
 
+The ANTLR grammar is intended to serve as the official reference grammar for OpenQASM3 and defines
+the set of syntactically valid statements in the language. ANTLR is used because it provides a
+human readable EBNF format which can evolve quickly. It further enables validation of the example
+files (see `openqasm/examples/`) via the built-in ANTLR parser. As an extension, it can be used
+to check whether a block of code is valid OpenQASM3.
```

Comment:
```suggestion
The ANTLR grammar is intended to serve as the official reference grammar for OpenQASM3 and defines
the set of syntactically valid statements in the language. ANTLR is used because it provides a
human-readable EBNF format that is computer-validatable and is used to validate the example
files (see `openqasm/examples/`) using the generated ANTLR parser. As an extension, it can be used
to check whether a source file is parseable OpenQASM3 (note this does not ensure that the OpenQASM3 is semantically correct).
```